htdocs/composer.json

```
@@ -0,0 +1,85 @@
+{
+	"name": "impresscms/impresscms",
+	"description": "ImpressCMS - A dynamic and user-friendly Content Management System",
+	"type": "project",
+	"keywords": ["cms", "content-management", "php", "mysql", "web"],
+	"homepage": "https://www.impresscms.org/",
+	"license": "GPL-2.0-or-later",
+	"authors": [
+		{
+			"name": "The ImpressCMS Project",
+			"homepage": "https://www.impresscms.org/",
+			"role": "Developer"
+		}
+	],
+	"require": {
+		"php": ">=7.4.0",
+		"ext-gd": "*",
+		"ext-json": "*",
+		"ext-mbstring": "*",
+		"ext-mysqli": "*",
+		"ext-pcre": "*",
+		"ext-pdo": "*",
+		"ext-session": "*",
+		"ext-xml": "*",
+		"ext-zlib": "*",
+		"composer/composer": "2.8.12",
```

plz don't add in core requirements version so explicity

```suggestion
		"composer/composer": "^2.8",
```

is fine